src/admin/hooks/useThumbnail.ts

```diff
@@ -29,7 +30,7 @@ const useThumbnail = (collection: SanitizedCollectionConfig, doc: Record<string,
     if (typeof adminThumbnail === 'function') {
       const thumbnailURL = adminThumbnail({ doc });
 
-      if (absoluteURLPattern.test(thumbnailURL)) {
+      if (absoluteURLPattern.test(thumbnailURL) || isBase64Image(thumbnailURL)) {
```

Comment:
Instead of importing a function, use a regex pattern just like on line 6.

```
const absoluteURLPattern = new RegExp('^(?:[a-z]+:)?//', 'i');
const base64Pattern = new RegExp('your_pattern_here');
```